lib/primer/classify/utilities.yml

```diff
@@ -1558,6 +1558,8 @@
 :text_transform:
   :uppercase:
   - text-uppercase
+  :capitalize:
```

Comment:
This file is auto-generated, so this change will be overwritten the next time the utilities script runs. It gets populated by the contents of custom_utilities.yml, so the change below is all you need 👍

Reply:
Gotcha, so should I remove this then?

Reply:
Yes please!

Reply:
Done!